src/app/UserService.ts

```diff
@@ -46,10 +46,29 @@ export class UserService {
       .promise();
   };
 
+  registerQuickActionsSectionVisibleFrom = async (
+    email: string,
+    date: Date
+  ): Promise<void> => {
+    await this._dbClient
+      .update({
+        TableName: this._userTableName,
+        Key: { email },
+        UpdateExpression:
+          "set quickActionsSectionVisibleFrom = :quickActionsSectionVisibleFrom",
+        ExpressionAttributeValues: {
+          ":quickActionsSectionVisibleFrom": date.toISOString(),
+        },
+      })
+      .promise();
+  };
```

Comment:
Is adding a test for this request possible?

Reply:
Yes, I agree with @andresmoschini about add a test for the new method.